src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -255,9 +258,17 @@ private string LoadContentFromResources(string name)
             }
 
             // Clean up possible script tags from document
-            return Regex.Replace(result, SCRIPT_TAG_REGEX, "");
+            if (Regex.IsMatch(result, SCRIPT_TAG_REGEX, RegexOptions.IgnoreCase))
+            {
+                LogWarning(Resources.ScriptTagsRemovalWarning, WarningLevel.Mild);
+                result = Regex.Replace(result, SCRIPT_TAG_REGEX, "", RegexOptions.IgnoreCase);
+            }
+
+            return result;
         }
 
+        private void LogWarning(string msg, WarningLevel level) => this.MessageLogged?.Invoke(LogMessage.Warning(msg, level));
```

Comment:
will this work in c# 6?

Reply:
It works. I know because I saw this PR's build go green having `OnLinkChanged` with a very similar implementation.